net/query_filter/query_filter.cc

```diff
@@ -133,3 +137,38 @@ absl::optional<GURL> ApplyQueryFilter(const GURL& original_url) {
   }
   return absl::nullopt;
 }
+
+bool ShouldFilterTwitterParam(
+    const GURL& url_to_load,
+    const absl::optional<url::Origin>& request_initiator) {
+  return url_to_load.DomainIs("twitter.com") &&
+         (!request_initiator ||
+          !request_initiator.value().DomainIs("twitter.com")) &&
+         url_to_load.has_query();
+}
+
+GURL RemoveTwitterParam(const GURL& url_to_load) {
+  GURL url = url_to_load;
+
+  std::vector<std::string> query_params =
+      base::SplitString(url.query_piece(), "&", base::KEEP_WHITESPACE,
+                        base::SplitResult::SPLIT_WANT_ALL);
+  query_params.erase(std::remove_if(query_params.begin(), query_params.end(),
```

Comment:
`base::EraseIf`

Reply:
Ah, I didn't realize `remove_if` was C++20 :) Thanks!

Reply:
`remove_if` is fine to use, but there's just a less error-prone alternative we have in Chromium codebase :)